index.bs

```diff
@@ -7245,6 +7284,33 @@ partial dictionary MLOpSupportLimits {
     1. Return |output|.
 </details>
 
+
+<div class="note">
+  The specific sampling algorithms are based on those widely used in existing Machine Learning frameworks. For example, when performing {{MLInterpolationMode/linear}} resampling from the following *[4, 4]* input tensor (considering only spatial dimensions):
+
+  ```
+  [   0   1   2   3  ]
+  [   0   1   2   3  ]
+  [  12  13  14  15  ]
+  [  12  13  14  15  ]
+  ```
+
+  For an *[8, 8]* output tensor, the expected values are:
+
+  ```
+  [   0   0.25   0.75   1.25   1.75   2.25   2.75   3  ]
+  [   0   0.25   0.75   1.25   1.75   2.25   2.75   3  ]
+  [   0   0.25   0.75   1.25   1.75   2.25   2.75   3  ]
+  [   3   3.25   3.75   4.25   4.75   5.25   5.75   6  ]
+  [   9   9.25   9.75  10.25  10.75  11.25  11.75  12  ]
+  [  12  12.25  12.75  13.25  13.75  14.25  14.75  15  ]
+  [  12  12.25  12.75  13.25  13.75  14.25  14.75  15  ]
+  [  12  12.25  12.75  13.25  13.75  14.25  14.75  15  ]
+  ```
+
+  This has the convenient properties that the sampling is evenly distributed and the corner values are aligned when up-sampling.
```

Comment:
It's also symmetric and robust to image mirroring. (no action - resolve me)

Reply:
We should add this text

Reply:
Added in 8a34038